CHANGELOG.md

```diff
@@ -8,6 +8,7 @@
 * Relax `RSpec/VariableDefinition` cop so interpolated and multiline strings are accepted even when configured to enforce the `symbol` style. ([@bquorning][])
 * Fix `RSpec/EmptyExampleGroup` to flag example groups with examples in invalid scopes. ([@mlarraz][])
 * Fix `RSpec/EmptyExampleGroup` to ignore examples groups with examples defined inside iterators. ([@pirj][])
+* Add `RSpec/MemoizedHelpersInExampleGroup` cop. ([@mockdeep][])
```

Comment:
Seems we have some inconsistency in the naming. Isn't it `MultipleMemoizedHelpers` ?

Reply:
Just spotted it as well 👍